OamLog.yaml

```diff
@@ -1994,6 +2076,54 @@ components:
                       type: string
                     logical-termination-point:
                       type: string
+    OamRequestRecord:
+      type: object
+      required:
+        - application-name
+        - release-number
+        - method
+        - resource
+        - stringified-body
+        - response-code
+        - user-name
+        - timestamp
+      properties:
+        application-name:
+          type: string
+          description: 'Name of application addressed by OaM request'
+        release-number:
+          type: string
+          pattern: ^([0-9]+)\.([0-9]+)\.([0-9]+)(?:-([0-9A-Za-z-]+(?:\.[0-9A-Za-z-]+)*))?(?:\+[0-9A-Za-z-]+)?$
```

Comment:
Can we update the release-number pattern to '^([0-9]{1,2})\.([0-9]{1,2})\.([0-9]{1,2})$' in "OamRequestRecord :: release-number"

Reply:
Yes, many thanks for indicating.
Changed as proposed.